host-spec/c06-consensus.tm

```diff
@@ -337,6 +423,25 @@
     there is no ambiguity about the underlying node.
   </definition>
 
+  At the end of each sync-epoch (def. <reference|defn-sync-epoch>) these
+  block arrivel times are used to update the current clock offset using the
+  median algorithm (alg. <reference|algo-slot-time>).
+
+  <\definition>
+    <label|defn-sync-epoch>A <with|font-series|bold|sync epoch> (unrelated to
```

Comment:
It would be nice to give an algorithm to determine the start of the sync epoch: e.g., store the last slot number s_i-1 in the previous epoch, check the difference between the slot number of last finalized block (s_i) and s_i-1 when BT is updated. If the difference is greater than or equal to s_{cq}, run output the  s_i

Reply:
Either that or simplifying the definition of a sync-epoch so far that this no longer becomes necessary.

Reply:
Addressed by simplifying the algorithm.